M2/Macaulay2/m2/packages.m2

```diff
@@ -557,6 +557,9 @@ package Dictionary := d -> (
     scan(unique prepend_currentPackage implicitlyLoadedPackages(),
 	pkg -> if pkg.Dictionary === d or pkg#"private dictionary" === d then break pkg))
 
+-- speeds up documentation generation by orders of magnitude
+package' = memoize package
```

Comment:
Is there a reason to keep `package` as the un-memoized version?

Reply:
No, except for the fact that `memoize` and methods don't interact well; see https://github.com/Macaulay2/M2/issues/3491#issuecomment-2714825592.